assets/js/anchor.js

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright 2018 Google LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+(function ($) {
+    'use strict';
+
+    // Headers' anchor link that shows on hover
+    $(function () {
+        // append anchor links to headings in markdown.
+        var article = document.getElementsByTagName('main')[0];
+        if (!article) {
+            return;
+        }
+        var headings = article.querySelectorAll('h1, h2, h3, h4, h5, h6');
```

Comment:
not using jQuery pattern because I'm not familiar with it. I've seen some other code here using native api too. So I assume this is fine. Let me know if you have concerns.

Reply:
Fine with me.